tests/dummy/app/pods/view/renderers/documentation/checkbox-array.md

```diff
@@ -29,3 +29,36 @@ Change what size checkboxes are used. See [ember-frost-core](http://ciena-frost.
   }
 }
 ```
+
+#### renderer.data
+
+Overrides the enum in the model to use specified label and value.
+
+```json
+{
+  "model": "foo",
+  "renderer": {
+    "name": "checkbox-array",
+    "data": [
+      {
+        "label": "The label for the checkbox",
+        "value": "The value for the checkbox"
+      }
+    ]
+  }
+}
+```
+
+#### renderer.selectedValues
+
+Allows checkboxes to be pre-checked. E.g. for retaining checks from query params upon refresh.
+
+```json
+{
+  "model": "foo",
+  "renderer": {
+    "name": "checkbox-array",
+    "selectedValues": ["values", "of", "selected", "checkboxes"]
```

Comment:
I'm not sure what you mean by "current value of that part of the model". 

I feel like this is pretty straight forward and simple to use and can be easily set by query params. See https://bitbucket.ciena.com/users/cc40498/repos/ciena-alarms-engine/browse/app/pods/alarms/modules/alarms-serializer.js#125-132 for the use case I am trying to provide. It's easy to simply take the query params and put them in there.

Are there a reasons why we wouldn't want it this way?
Don't get me wrong, if it is bad I am fine with changing it.